### PR TITLE
update ruRU

### PR DESCRIPTION
**Pull request by andaril**
`L.infobox_stacks = "%d |4Stack:Stacks;: %d |4player:players;" -- 4 Stacks: 5 players // 1 Stack: 1 player`

I'm not sure how to use this. In russian we have 2, 3, 4, 22, 23, etc countables for word `игрока`(players) and 5-20 for `игроков`(players) and finally 21 - `игрок`(player)

---------------

`-- Abbreviated numbers`
You or I could revert this, but this translation abbr is more native for us.

### CODE REVIEW

**Comment by funkydude** [comment]
@andaril Thanks!

>I'm not sure how to use

You can add more words. Hope this helps: https://github.com/BigWigsMods/BigWigs/blob/dcee8f252c3e98fd46ca82883cdba0f02f385f71/Plugins/Locales/ruRU.lua#L142

File: CastleNathria/Locales/ruRU.lua [hunk 1]
```diff
@@ -28,75 +28,75 @@ L = BigWigs:NewBossLocale("Lady Inerva Darkvein", "ruRU")
...
+	L.dance_assist_left = "|T450906:0:0:0:0:64:64:4:60:4:60|t Двигайся влево |T450906:0:0:0:0:64:64:4:60:4:60|t"
 	-- These need to match the in-game boss yells
-	--L.dance_yell_up = "Forward" -- Prance Forward!
-	--L.dance_yell_right = "right" -- Shimmy right!
-	--L.dance_yell_down = "down" -- Boogie down!
-	--L.dance_yell_left = "left" -- Sashay left!
+	L.dance_yell_up = "вперед" -- Prance Forward!
+	L.dance_yell_right = "вправо" -- Shimmy right!
+	L.dance_yell_down = "назад" -- Boogie down!
+	L.dance_yell_left = "влево" -- Sashay left!
```
Comment (funkydude) [inline CastleNathria/Locales/ruRU.lua:78]: Could you add the full yell in the comments here for future reference?

Comment (andaril) [reply CastleNathria/Locales/ruRU.lua]: fixed

**Comment by andaril** [comment]
Fixed all.

Not translated part of Denathrius is on mythic difficulty I think. I hope I or somebody else will translate it soon. 
I'll wait for videos of russian kill with open chat frame. 

Two years ago was an addon that records boss chat and abilities. It's still exists? Could't find it.

**Comment by funkydude** [comment]
Thanks! It's called [Transcriptor](https://www.curseforge.com/wow/addons/transcriptor) and is still available. It's what we use even today.

**Pull request merged**